packages/bundle/src/createCognitiveServicesSpeechServicesPonyfillFactory.js

```diff
@@ -63,6 +64,28 @@ export default function createCognitiveServicesSpeechServicesPonyfillFactory({
 
     const source = audioConfig.privSource;
 
+    // HACK: In Speech SDK 1.12.0-1.13.1, it dropped support of macOS/iOS Safari.
```

Comment:
I don't know if 'hack' is a policheck violation or not, but to be safe, please replace with WORKAROUND